README.md

```diff
@@ -277,6 +277,14 @@ sequencer.insertSteps(index,module_name,optional_options);
 
 return value: **`sequencer`** (To allow method chaining)
 
+###Importing an independent module
```

Comment:
Strict markdown requires a space after `###` -- `### Importing...`

Also, to keep things tidy, let's leave a blank line after this one.

Reply:
done!